src/app/member/components/MemberDetials.test.tsx

```diff
@@ -0,0 +1,23 @@
+import { render, screen } from '@testing-library/react';
+import userEvent from "@testing-library/user-event";
+import { it, describe, expect} from 'vitest'; 
+import { MemberDetials } from './MemberDetials';
+
+describe('ActionsMenu Component', () => {
+  it('calls the correct action when an item is clicked', async () => {
+    render(<MemberDetials userInfo={{
```

Comment:
can we create a separate object and pass it to here,because due to this code will be clean and we can use that object in whole testing for future testing (if further testing need)

Reply:
fixed